cfme/automate/dialogs/dialog_box.py

```diff
@@ -29,6 +28,7 @@ class Box(BaseEntity):
     box_label = attr.ib()
     box_desc = attr.ib(default=None)
 
+    from cfme.automate.dialogs.dialog_element import ElementCollection
```

Comment:
Out of curiosity, are we using all these local imports to avoid circular dependencies?

Reply:
The path to the import won't impact whether or not the import is circular, but _where_ they are imported will. There are a lot of cross dependencies in these dialog modules, which is why imports are placed within the classes/methods.